styleguide/source/_patterns/03-templates/04-topic/topic.twig

```diff
@@ -0,0 +1,17 @@
+<header>
+  {% include 'organisms-ribbon' %}
+</header>
+
+<main class="topic" style="display: flex; flex-direction: row;">
```

Comment:
Suggestion to remove or move inline CSS in this file into a new Scss sheet in the Topic directory.